apps/fluent-tester/src/FluentTester/TestComponents/ContextualMenu/ContextualMenuTest.tsx

```diff
@@ -234,6 +360,10 @@ const contextualMenuSections: TestSection[] = [
     name: 'Nested ContextualMenu',
     component: nestedContextualMenu,
   },
+  {
+      name: "IconButton with ContextualMenu",
```

Comment:
This should be in single quotes. Do you have the ESLint extension on Visual Studio Code? ESLint should catch this automatically.